nixos/modules/services/admin/docuum.nix

```diff
@@ -0,0 +1,37 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.docuum;
+in
+{
+  options.services.docuum = {
+    enable = mkEnableOption "docuum daemon";
+
+    threshold = mkOption {
+      description = "Threshold for deletion";
+      type = types.str;
+      default = "10 GB";
+    };
+  };
+
+  config = mkIf (cfg.enable) {
+    assertions = [
+      {
+        assertion = config.virtualisation.docker.enable;
+        message = "docuum requires docker on the host";
+      }
+    ];
+
+    systemd.services.docuum = {
+      after = [ "docker.socket" ];
+      requires = [ "docker.socket" ];
+      wantedBy = [ "multi-user.target" ];
+      path = [ config.virtualisation.docker.package ];
+      serviceConfig = {
+        ExecStart = "${pkgs.docuum}/bin/docuum --threshold ${escapeShellArg cfg.threshold}";
```

Comment:
Use `lib.getExe` and `utils.escapeSystemdExecArgs` by making the params and arguments into a list.